sdk/openai/openai/src/OpenAIClient.ts

```diff
@@ -130,18 +130,32 @@ export class OpenAIClient {
     options: GetAzureBatchImageGenerationOperationStatusOptions = {
       requestOptions: {},
     }
-  ): Promise<BatchImageGenerationOperationResponse> {
+  ): Promise<ImageGenerationResponse> {
     return getAzureBatchImageGenerationOperationStatus(this._client, operationId, options);
   }
 
   /** Starts the generation of a batch of images from a text caption */
   beginAzureBatchImageGeneration(
     prompt: string,
-    options: BeginAzureBatchImageGenerationOptions = { requestOptions: {} }
-  ): Promise<BatchImageGenerationOperationResponse> {
+    options: ImageGenerationOptions = { requestOptions: {} }
+  ): Promise<ImageGenerationResponse> {
     return beginAzureBatchImageGeneration(this._client, prompt, options);
   }
 
+  /**
+   * Starts the generation of a batch of images from a text caption 
+   * @param prompt - The prompt to use for this request.
+   * @param options - The options for this image request.
+   * @returns The image generation response (containing url or base64 data).
+   */
+  getImage(
```

Comment:
if this is just an alias of `beginAzureBatchImageGeneration`, should we simply rename the other method? 

Also in the future I imagine `beginAzureBatchImageGeneration` will return a proper `Poller<T>` being an LRO, which is why it is prefixed with `begin` as a hint that it is not a normal async operation. Will this method also adopt that contract or is this one supposed to block until the batch is complete?

If the latter, perhaps this convenience method should have more logic to handle checking the status and returning the final result?

Reply:
> if this is just an alias of `beginAzureBatchImageGeneration`, should we simply rename the other method?

1. I asked this exact question (in the AOAI teams chat that I just added you to) about renaming and was told to alias it. According to @trrwilson:
"- Non-Azure isn't a batch operation at all, so that'd be misleading
 - We'll replace Azure's use of "batch" (LRO) with the new non-batch version as soon as it becomes available (another service API version or two)"

> Also in the future I imagine `beginAzureBatchImageGeneration` will return a proper `Poller<T>` being an LRO, which is why it is prefixed with `begin` as a hint that it is not a normal async operation. Will this method also adopt that contract or is this one supposed to block until the batch is complete?

Apparently the underlying operation is moving to non-LRO Soon(tm).

Reply:
Interesting! And here I thought "batch" referred to the fact we were generating multiple images rather than it being an LRO. Whoops.

If the non-Azure version isn't an LRO, does this method work at all? I wasn't able to test against the openai.com endpoint due to a lack of credentials. 

I also wonder if we should paper over the Azure LRO version now rather than exposing it today and then getting rid of it in a later service version.

Reply:
Makes sense, updated API name